addons/hr_timesheet/models/project.py

```diff
@@ -80,18 +80,26 @@ def _search_is_internal_project(self, operator, value):
             operator_new = 'not inselect'
         return [('id', operator_new, (query, ()))]
 
-    @api.depends('allow_timesheets', 'task_ids.planned_hours', 'task_ids.remaining_hours')
+    @api.depends('allow_timesheets', 'task_ids.planned_hours', 'timesheet_ids')
     def _compute_remaining_hours(self):
-        group_read = self.env['project.task'].read_group(
+        group_read_effective_hours = self.env['account.analytic.line'].read_group(
+            domain=[('project_id', 'in', self.filtered('allow_timesheets').ids),
+                    ('task_id', '!=', False), ('task_id.planned_hours', '!=', False),
+                    '|', ('task_id.stage_id.fold', '=', False), ('task_id.stage_id', '=', False)],
```

Comment:
Why not taking those tasks into account, we worked on them didn't we?
```suggestion
```

Reply:
It's the same logic as the `planned_hours`; we do not want to have reaming time of tasks that are in `Done` stage (should be folded) for example, right?

Reply:
`planned_hours` is a normal stored field isn't it?

Reply:
it is :slightly_smiling_face: